LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -49,7 +49,11 @@
  Any help would be greatly appreciated.
  */
 
+#if defined(LTH_APP_EXTENSIONS)
```

Comment:
Hi, yep, the extensions are additional targets in the project, so their setup may be different from the main app. I added the macro like they do in SVProgressHud and other projects: https://github.com/SVProgressHUD/SVProgressHUD#app-extensions
So, in the target of the main app everything is the same, and the target of the extension has LTH_APP_EXTENSIONS=1 under preprocessor macros ("build settings" tab).